components/root-actions/AccountSettings.js

```diff
@@ -80,7 +94,38 @@ const AccountSettings = () => {
                   />
                 </Box>
               )}
+              <Box>
+                <StyledCheckbox
+                  name="2FA"
+                  label="2FA"
+                  disabled={!twoFactorEnabledFlag && !enableSave}
+                  checked={twoFactorEnabledFlag}
+                  onChange={({ checked }) => {
+                    setEnableSave(true);
+                    setTwoFactorEnabledFlag(checked);
+                  }}
+                />
+              </Box>
             </Flex>
+            {!twoFactorEnabledFlag && enableSave && (
+              <Container pt={4}>
+                <MessageBox type="warning" withIcon>
+                  <FormattedMessage
+                    defaultMessage={`Make sure to take appropriate steps in order verify the user before resetting the 2FA codes. Reference: {reference}`}
+                    values={{
+                      reference: (
+                        <StyledLink
+                          href="https://github.com/opencollective/opencollective/issues/4478#issuecomment-887483590"
+                          openInNewTab
+                        >
+                          https://github.com/opencollective/opencollective/issues/4478#issuecomment-887483590
+                        </StyledLink>
```

Comment:
Great that you've included this message, but could we display the policy in-line rather than linking to the Github comment?

Reply:
Also, we could add "In doubt, do not hesitate to ask the engineering team".

Reply:
And the more flashy it is, the better. This message should ideally burn the eyes of anyone willing to use the feature!

Reply:
How does this look. I've made it type `error` so that the red looks more in the face; 

![image](https://user-images.githubusercontent.com/12435965/216626928-cf146cd4-0036-47b5-9af7-12e36f825a79.png)